cephprimarystorage/cephprimarystorage/cephagent.py

```
@@ -716,13 +715,15 @@ def retry(times=3, sleep_time=3):
             def wrap(f):
                 @functools.wraps(f)
                 def inner(*args, **kwargs):
+                    ex = None
                     for i in range(0, times):
```

Ruff (0.8.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)